test/evergreen.yml

```diff
@@ -5043,6 +5043,88 @@ tasks:
         vars:
           test-name: checkpoint-stress.wtperf
 
+  - name: perf-test-live-restore
+    tags: ["long-perf"]
+    patch_only: true
+    depends_on:
+      - name: perf-test-long-500m-btree-populate
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 10800
+          timeout_secs: 10800
+      # Fetch the compile artifacts.
+      - func: "fetch artifacts"
+      # Fetch the database created by perf-test-long-500m-btree-populate task. This will be used
+      # as the source for live restore.
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: perf-test-long-500m-btree-populate
+          destination: "wiredtiger/cmake_build/bench/wtperf/SOURCE"
+      # Create an empty home directory to be used as the destination for live restore.
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build/bench/wtperf"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            mkdir WT_TEST_0_0
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: 500m-btree-50r50u-live-restore.wtperf
+          maxruns: 1
+          no_create: true
+          wtarg: -args ['"-C create,statistics=(fast),statistics_log=(json,wait=1,sources=[file:]),live_restore=(enabled=true,path=SOURCE)"'] -ops ['"read", "update", "warnings", "max_latency_read_update"']
+      - func: "upload stats to atlas"
+        vars:
+          test-name: 500m-btree-50r50u-live-restore.wtperf
+      - func: "upload stats to evergreen"
+        vars:
+          test-name: 500m-btree-50r50u-live-restore.wtperf
+      - func: "cleanup"
+      
+  - name: perf-test-live-restore-no-server
```

Comment:
Given there is no obvious control task, i.e. no live restore already. I assume that already exists and runs the same populate followed by `500m-btree-50r50u-live-restore-no-server.wtperf` stage? 

If so, how do we compare the results? Do you need to open up 3 different tasks in evergreen?

Reply:
Yes at the moment you need to open the 3 different tasks for comparison:
- perf-test-long-500m-btree-50r50u
- perf-test-long-500m-btree-50r50u-live-restore
- perf-test-long-500m-btree-50r50u-live-restore-no-server

We could plot these on the same chart in our dashboard though?